packages/authorization-process/test/createApiClient.test.ts

```diff
@@ -0,0 +1,77 @@
+import { describe, it, vi, beforeAll, afterAll, expect } from "vitest";
+import {
+  Client,
+  ClientAddedV2,
+  UserId,
+  clientKind,
+  generateId,
+  toClientV2,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { genericLogger } from "pagopa-interop-commons";
+import {
+  decodeProtobufPayload,
+  readLastEventByStreamId,
+} from "pagopa-interop-commons-test/index.js";
+import { ApiClientSeed } from "../src/model/domain/models.js";
+import { authorizationService, postgresDB } from "./utils.js";
+
+describe("createConsumerClient", () => {
+  const userId = generateId();
+
+  beforeAll(async () => {
+    vi.useFakeTimers();
+    vi.setSystemTime(new Date());
+  });
+
+  afterAll(() => {
+    vi.useRealTimers();
+  });
+
+  const clientSeed: ApiClientSeed = {
+    name: "Seed name",
+    description: "Description",
+    members: [userId],
+  };
+  it("should write on event-store for the creation of a api client", async () => {
+    const { client } = await authorizationService.createApiClient(
+      clientSeed,
+      unsafeBrandId(userId),
+      generateId(),
+      genericLogger
+    );
+
+    const writtenEvent = await readLastEventByStreamId(
+      client.id,
+      '"authorization"',
+      postgresDB
+    );
+
+    expect(writtenEvent).toMatchObject({
+      stream_id: client.id,
+      version: "0",
+      type: "ClientAdded",
+      event_version: 2,
+    });
+
+    const writtenPayload = decodeProtobufPayload({
+      messageType: ClientAddedV2,
+      payload: writtenEvent.data,
+    });
+
+    const expectedClient: Client = {
+      id: client.id,
+      keys: [],
+      name: clientSeed.name,
+      createdAt: new Date(),
+      consumerId: unsafeBrandId(userId),
+      kind: clientKind.api,
+      purposes: [],
+      relationships: [],
+      users: clientSeed.members.map(unsafeBrandId<UserId>),
```

Comment:
```suggestion
      users: [unsafeBrandId<UserId>(userId)],
```
In the tests let's avoid to introduce the logic we wish to test, we can just insert here what we expect